script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -255,9 +261,10 @@ Greater Dead Apostles do not necessarily show absolute obedience toward their pa
 [sha:3657a1217676c255d6edaf210d2cfbf64f2019c7]{
 -- Page 593, Offset 18060.
 -- まあでも、祖を倒す事はほぼ不可能よ。同じ『自由』を得たとしても祖が生きた年月は膨大だもの。上級死徒では祖の存在規模を上回ることは難しい」
-That said, taking down an Ancestor is no mean feat.
- Even if you acquire the same 'freedoms', the amount of time they've spent on this earth is colossal in comparison.
- It's difficult for a Greater Dead Apostle to surpass the Lifescale of an Ancestor."
+// 存在規模 previously translated as <power/abilities|Lifescale>
+That said, taking down an Ancestor is practically impossible.
+ Even if they've acquired the same 'freedoms', Ancestors have in comparison lived far longer.
+ Greater Dead Apostles have a hard time surpassing the Lifescale of an Ancestor."
```

Comment:
I kinda prefer the original here, though the last sentence on the new ver might be the better one there